### PR TITLE
Updated README for versioning strategy

### PR DESCRIPTION
**Pull request by TechnotronicOz**
Please create a tag from this branch for `v1.0`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -268,7 +268,7 @@ Installation
 
 To install Testify, use `go get`:
 
-    go get github.com/stretchr/testify
```
Comment (ernesto-jimenez) [inline README.md]: I would offer this as an alternative rather than the default.

**Comment by TechnotronicOz** [comment]
@ernesto-jimenez Should be updated

**Comment by TechnotronicOz** [comment]
@ernesto-jimenez Any updates?

**Comment by ernesto-jimenez** [comment]
@DAddYE looks good to you?

**Comment by DAddYE** [comment]
:+1:

File: README.md [hunk 2]
```diff
@@ -268,7 +268,8 @@ Installation
...
+    * Latest version: go get github.com/stretchr/testify
+    * Specific version: go get gopgk.in/stretchr/testify.v1
```
Comment (ernesto-jimenez) [inline README.md]: Just realised there's a typo: it's `gopkg.in`

**Comment by TechnotronicOz** [comment]
@ernesto-jimenez Good catch, my apologies for the typo. It is updated.

**Comment by TechnotronicOz** [comment]
@ernesto-jimenez Any updates?

**Comment by ernesto-jimenez** [comment]
Sorry, busy week

**Comment by TechnotronicOz** [comment]
@ernesto-jimenez No worries, thanks!

**Comment by ernesto-jimenez** [comment]
My pleasure. Hope that helps :)

**Pull request merged**